test/regression_tests.sh

```diff
@@ -28,9 +28,6 @@ install_torchserve_from_source() {
   cd serve
   echo "Installing torchserve torch-model-archiver from source"
   ./scripts/install_from_src_ubuntu
-  echo "TS Branch : " $(git rev-parse --abbrev-ref HEAD) >> $3
-  echo "TS Branch Commit Id : " $(git rev-parse HEAD) >> $3
-  echo "Build date : " $(date) >> $3
   echo "Torchserve Succesfully installed"
```

Comment:
This info was recently added to track date / branch / commit against which the test ran. Please dont remove this or add this elsewhere.

Reply:
updated as required